__tests__/String/head.test.ts

```diff
@@ -0,0 +1,10 @@
+import { S, O } from '../..'
+
+ describe('head', () => {
+   it('returns None if the given string is empty.', () => {
+     expect(S.head('')).toEqual(O.None)
+   })
+   it('Returns Some(value) where value is the first character of the string', () => {
+     expect(S.head('head-test')).toEqual('h')
+   })
+ })
```

Comment:
(and again 😅) could you please add a test with the `*` string as the first argument of `it`?

Reply:
😅